src/backend/mi2/mi2.ts

```diff
@@ -883,7 +883,7 @@ export class MI2 extends EventEmitter implements IBackend {
 	async getRegisterValues(): Promise<RegisterValue[]> {
 		if (trace)
 			this.log("stderr", "getRegisterValues");
-		const result = await this.sendCommand("data-list-register-values N");
+		const result = await this.sendCommand("data-list-register-values N"+ " " +this.registerLimit);
```

Comment:
`const result = await this.sendCommand("data-list-register-values N " + this.registerLimit);`
but maybe we should use
`const result = await this.sendCommand("data-list-register-values --skip-unavailable N " + this.registerLimit);`?